content/en/docs/reference/instrumentation/slis.md

```diff
@@ -0,0 +1,91 @@
+---
+reviewers:
+- logicalhan
+title: Kubernetes Component SLIs
+content_type: reference
+weight: 20
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.26" state="alpha" >}}
+
+The Component SLIs feature allows enabling of a new metrics endpoint for each Kubernetes
+binary. This metric endpoint is exposed on the standard port of each binary at the endpoint
+`metrics/slis`. 
+
+<!-- body -->
+
+## SLI Metrics
+
+We expose two metrics; a gauge (which represents the current state of the healthcheck) and a
+counter (which records the cumulative counts that we've observed for each healthcheck state).
+
+```golang
+var (
+   // healthcheck is a Prometheus Gauge metrics used for recording the results of a k8s healthcheck.
+   healthcheck = k8smetrics.NewGaugeVec(
+      &k8smetrics.GaugeOpts{
+         Namespace:      "kubernetes",
+         Name:           "healthcheck",
+         Help:           "This metric records the result of a single healthcheck.",
+         StabilityLevel: k8smetrics.ALPHA,
+      },
+      []string{"name", "type"},
+   )
+
+   // healthchecksTotal is a Prometheus Counter metrics used for counting the results of a k8s healthcheck.
+   healthchecksTotal = k8smetrics.NewCounterVec(
+      &k8smetrics.CounterOpts{
+         Namespace:      "kubernetes",
+         Name:           "healthchecks_total",
+         Help:           "This metric records the results of all healthcheck.",
+         StabilityLevel: k8smetrics.ALPHA,
+      },
+      []string{"name", "type", "status"},
+   )
+)
```

Comment:
It's OK to omit the source code; we don't expect readers to know how to code in Golang. We could hyperlink to the implementation if that's important.

Please either omit this or make it clear why we should include it.

Reply:
I actually do expect people who interact with prometheus metrics to be able to read the metric definitions. It's mostly declarative and easier to parse than most random code.

Reply:
removed